bedrock/mozorg/templates/mozorg/contribute/friends.html

```diff
@@ -59,6 +78,14 @@
         <div class="field field-country">{{ newsletter_form.country|safe }}</div>
         <div class="field field-format">{{ newsletter_form.fmt|safe }}</div>
         <div class="field field-privacy">{{ newsletter_form.privacy|safe }}</div>
+        <div class="field field-fx-and-you">
+          <label for="id_fx-and-you">
+            <input id="id_fx-and-you" name="fx-and-you" type="checkbox">
+            <span class="title">
+              {{ _('Yes, I’d like to stay up-to-date with all the latest and greatest Firefox news by subscribing to the Firefox + You newsletter here.') }}
```

Comment:
Nit: Worth placing some emphasis on the name of the newsletter in this line of copy? `<em>Firefox + You</em>`